src/ToricVarieties/NormalToricVarieties/methods.jl

```diff
@@ -30,10 +30,10 @@ function betti_number(v::AbstractNormalToricVariety, i::Int)
             pushfirst!(f_vector, 1)
             betti_numbers[i+1] = fmpz(sum((-1)^(i-k) * binomial(i,k) * f_vector[dim(v) - i + 1] for i=k:dim(v)))
         end
-        set_attribute!(v, :betti_number, betti_numbers)
+        set_attribute!(v, :betti_numbers, betti_numbers)
     end
     
     # return result
-    return get_attribute(v, :betti_number)[i+1]
+    return get_attribute(v, :betti_numbers)[i+1]
```

Comment:
Better (more efficient and type stable):
```suggestion
    return betti_numbers[i+1]
```